changelog/unreleased/fix-deleting-during-postprocessing.md

```diff
@@ -0,0 +1,7 @@
+Bugfix: Disallow to delete a file during the processing
+
+We want to disallow deleting a file during the processing to prevent collecting the orphan uploads.
+
+https://github.com/owncloud/ocis/pull/8131
```

Comment:
This is pointing to a completely unrelated issue. I guess you wanted it to point to the reva bump that included your change? #8207 maybe?

Reply:
Thanks @rhafer, the pr number should be [#8132](https://github.com/owncloud/ocis/pull/8132) and reva changes already included in a bump which you mentioned